src/components/LastDeliveredCell.tsx

```diff
@@ -0,0 +1,30 @@
+import { DefaultServerCellComponentProps } from 'payload'
+import { getTimeAgo } from '@/lib/utils'
+
+const LastDeliveredCell = async ({ rowData, payload, collectionSlug }: DefaultServerCellComponentProps) => {
+  const customerId = collectionSlug === 'customers' ? rowData.id : rowData.customer; 
+  const transactionAt = (await payload.find({
```

Comment:
instead of `transactionAt` we can call `lastTransactionAt`
```suggestion
  const lastTransactionAt = (await payload.find({
```